### PR TITLE
Css brochure form

### PR DESCRIPTION
**Pull request by EuichangPark**
brochurefinal

### CODE REVIEW

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]
Getting there. The form does need some tweaks to look more like the example:
- The form labels need to use the right font family, font weight, and indenting.
- The form controls have their background colors flipped and need more light-weight borders.
- The placeholders need more spacing around them.
- The submit button needs a higher font weight.

---

image

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]
@EuichangPark Please add an updated screenshot to your pull request.

**Comment by EuichangPark** [comment]
finalbrochureform

File: css-brochure-form/index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+<body>
+  <form id="learnMore">
+    <h2>LEARN MORE</h2>
+    <div class="comEmail">
+      <div>
+        <label for="compEmail" id="companyemail">COMPANY EMAIL</label>
+      </div>
+      <input id="compEmail" type="text" placeholder="Kent@gmail.com" required>
+    </div>
+
```
Comment (Zhiwutian) [inline css-brochure-form/index.html:22]: remove this extra whitespace in the document!

**Review by Zhiwutian — APPROVED** [review]
Approved with comment!

**Pull request merged**